ami/ml/orchestration/jobs.py

```
@@ -0,0 +1,102 @@
+        successful_queues = 0
+        failed_queues = 0
+
+        async with TaskQueueManager() as manager:
+            for image_pk, task in tasks:
+                try:
+                    logger.info(f"Queueing image {image_pk} to stream for job '{job.pk}': {task.image_url}")
+                    success = await manager.publish_task(
+                        job_id=job.pk,
+                        data=task,
+                    )
+                except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)